codespell_lib/data/dictionary.txt

```diff
@@ -6175,25 +6278,41 @@ instansiate->instantiate
 instansiated->instantiated
 instansiates->instantiates
 instansiation->instantiation
+instantations->instantiations
+instantiaties->instantiates
+instanze->instance
 instatance->instance
 insted->instead
+inster->insert
+insterad->instead
 instersction->intersection
 instert->insert
 insterted->inserted
+instertion->insertion
 institue->institute
+instnce->instance
+instrace->instance
+instread->instead
 instruciton->instruction
 instrucitons->instructions
 instrumentaion->instrumentation
+instsallation->installation
 instuction->instruction
 instuments->instruments
 instutionalized->institutionalized
 instutions->intuitions
+insufficent->insufficient
+insuffiency->insufficiency
 insurence->insurance
+intaces->instance
 intall->install
+intance->instance
```

Comment:
`intance` already exists

https://travis-ci.org/lucasdemarchi/codespell/jobs/376870882#L670

It's probably worth running `pytest codespell_lib` locally so you don't have to wait for Travis to give this feedback

Reply:
how do I run `pytest codespell_lib` ?  
`bash: pytest: command not found`

Reply:
you need to install the `pytest` and `pytest-cov` Python packages